nextcord/state.py

```diff
@@ -2367,6 +2367,40 @@ def parse_guild_audit_log_entry_create(self, data) -> None:
                 user,
             )
 
+    def parse_message_poll_vote_add(self, data) -> None:
+        message = self._get_message(int(data["message_id"]))
+        answer_id = data["answer_id"]
+        if message:
+            user = self.get_user(int(data["user_id"]))
+            self.dispatch(
+                "message_poll_vote_add",
+                message,
+                answer_id,
+                user,
+            )
+        else:
+            _log.debug(
+                "message_poll_vote_add was not dispatched because the message (%r) is None",
```

Comment:
There should be a raw event in this case (both events), as messages aren't cached for too long.